src/main/java/com/uid2/admin/vertx/service/EncryptionKeyService.java

```diff
@@ -551,6 +551,15 @@ private int getOrCreateKeysetId(int siteId)
         AdminKeyset keyset = lookUpKeyset(siteId, currentKeysets);
         if(keyset == null) {
             int newKeysetId = getMaxKeyset(currentKeysets)+1;
+            if(siteId == Const.Data.MasterKeySiteId) {
```

Comment:
where are the calls to getOrCreateKeysetId() that will create these special keysets?